src/dbos-executor.ts

```diff
@@ -2175,10 +2186,9 @@ export class DBOSExecutor implements DBOSExecutorContext {
     return merged;
   }
 
-  async resumeWorkflow(workflowID: string): Promise<WorkflowHandle<unknown>> {
-    await this.systemDatabase.resumeWorkflow(workflowID);
+  async resumeWorkflow(workflowID: string) {
```

Comment:
Why are we changing the interface here?  Shouldn't this still return a workflow handle?